Common/GPU/Vulkan/VulkanRenderManager.h

```diff
@@ -291,6 +291,8 @@ class VulkanRenderManager {
 
 	void SetScissor(VkRect2D rc) {
 		_dbg_assert_(curRenderStep_ && curRenderStep_->stepType == VKRStepType::RENDER);
+		_dbg_assert_((int)rc.offset.x >= 0);
```

Comment:
I assume one could use the display layout editor to make these values negative (i.e. CenterDisplayOutputRect)?

-[Unknown]

Reply:
Hm, yeah, good point - though it technically is the callers responsibility to do the clipping, maybe it indeed makes sense to tolerate bad dimensions here and adjust automatically instead of asserting.